flowredux/src/commonMain/kotlin/com/freeletics/flowredux/sideeffects/InStateSideEffectBuilder.kt

```diff
@@ -16,6 +19,18 @@ internal abstract class InStateSideEffectBuilder<InputState : S, S, A> {
 
     abstract fun generateSideEffect(): SideEffect<S, A>
 
+    protected inline fun stateChange(
```

Comment:
I personally would find the wording `changeState()` a bit more readable than `stateChange()` but your call, it doesn't make code-wise any difference.

Reply:
Updated [670fb26](https://github.com/freeletics/FlowRedux/pull/504/commits/670fb2689c2d12eb12c46d33a6c1fa08fe4eb578)